src/eradiate/scenes/phase/_blend.py

```diff
@@ -199,7 +211,7 @@ def kernel_dict(self, ctx: KernelDictContext) -> KernelDict:
         if self.weights.ndim == 2 and self.weights.shape[1] > 1:
             write_binary_grid3d(
                 filename=self.weight_file,
-                values=weights[np.newaxis, np.newaxis, ...],
+                values=weight[np.newaxis, np.newaxis, ...],
```

Comment:
Let's take the opportunity to use a nicer syntax:
```suggestion
                values=np.reshape(weight, (1, 1, -1)),
```